lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -927,23 +927,20 @@ class SkPaint {
 
 @JS()
 @anonymous
+@staticInterop
 abstract class CkFilterOptions {}
 
 @JS()
 @anonymous
+@staticInterop
 class _CkCubicFilterOptions extends CkFilterOptions {
-  external double get B;
```

Comment:
What errors were you getting from moving these to extensions? Are these unused (similar question for the other removed members)?

Reply:
Yea, they appear to be unused and so the analyzer warned. The flutter web project has a check that turns the analyzer warnings to errors. It sort of makes sense that the analyzer is stricter with extension methods because they are static.

I'm not sure what we can really do here other than maybe some annotation that makes the warnings go away, but given that the methods really are seemingly unused, it may make sense to just remove them.

Reply:
Agreed, I was just mostly curious why warnings started to appear now instead of before when they were unused anyways.